citeproc-java/templates/Parser.java

```diff
@@ -80,8 +80,11 @@
 				builder = null;
 				continue;
 			}
+			if (line.isEmpty()) { 
+				continue;    // allow for empty lines if the separator is not empty? This is wrong format, but exported thus by Zotero
+			}
 			
-			if (line.length() < ${valuePos + 1}) {
+			if ((line.length()>0) && (line.length() < ${valuePos + 1})) {  // allow for empty lines
```

Comment:
This change is not necessary. `line.length()` should always be greater than 0 here.